### DISCUSSION TITLE
problems with doi

### DISCUSSION THREAD

**Discussion by Nfnd** [knowledgefutures/pubpub#3013] [Support] [answered]
our landscape report is published on Dec 7 and still doi is not working

the release: https://oabb.pubpub.org/pub/open4de/release/1

https://doi.org/10.21428/986c5d43.bab38f02

I have reported it many times and I have updated the deposit via pub settings.  is there any way to resolve this problem? What else can I do? And why doesn't it work?

Thanks for your help!

![Screenshot 2024-02-12 093802](https://github.com/pubpub/pubpub/assets/80816254/32d4f60b-c9de-4716-828c-46630c4c45eb)

**Comment by gabestein** [comment, marked as answer]
Hello — this was being caused by contributor ORCIDs being entered with the wrong format for our database (though that isn't readily apparent to the user and I filed a fix for it here #3014). I've re-submitted your DOI and it appears to work.

**Reply by Nfnd** [reply]
Thanks so much, it is working now!
